server/utils/db/db_utils.py

```
@@ -2986,6 +2999,17 @@ def initialize_tables():
                 logging.warning(f"Error adding expires_at to org_invitations: {e}")
                 conn.rollback()
 
+            # Migration: Add onboarding_completed column to organizations table
+            try:
+                cursor.execute(
+                    "ALTER TABLE organizations ADD COLUMN IF NOT EXISTS onboarding_completed BOOLEAN DEFAULT FALSE;"
+                )
+                conn.commit()
+                logging.info("Ensured onboarding_completed column exists on organizations table.")
+            except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)